src/dealDashboard/dealDiscussion/discussionsList/discussionsList.html

```diff
@@ -52,7 +54,7 @@ <h1 class="subtitle heading heading1 gradient" id="discussionsSection">
             </p>
           </div>
           <div data-test="number-of-replies" class="td h-centered v-centered">${discussion.replies || 0}</div>
-          <div class="td h-centered v-centered">${discussion.lastModified}</div>
+          <div class="td h-centered v-centered">${discussion.modifiedAt | dateDiff}</div>
```

Comment:
I suspect there are other places we could use this valueconverter.  If so, can you find them and apply it in those places too?

Reply:
Done: I only limited search within discussions for now.

https://github.com/PrimeDAO/prime-deals-dapp/pull/653/commits/b4cf547c76b794167d05fc1203b6f4aa7986a85b